packages/transport-http/src/send-ping.ts

```diff
@@ -0,0 +1,35 @@
+import {invariant} from "@onflow/util-invariant"
+import {httpRequest as defaultHttpRequest} from "./http-request.js"
+import { IIx } from "@onflow/typedefs"; 
+
+export interface IContext {
+  response?: Function;
+}
+
+export interface IOpts {
```

Comment:
I think these should be named more explicitly.  It doesn't seem great to have to alias import these every time they used.